backend/README.md

```diff
@@ -1,15 +1,20 @@
 # vault-service
 
-To install dependencies:
+## Getting started
+
+**Install dependencies**
 
 ```bash
 bun install
 ```
 
-To run:
+**Run the server**
 
 ```bash
 bun run index.ts
 ```
 
-This project was created using `bun init` in bun v1.0.18. [Bun](https://bun.sh) is a fast all-in-one JavaScript runtime.
+**Updating the database schema**
+
+1. Update the tables in `src/db/schema.ts`.
+2. Run `npx drizzle-kit generate:pg` to generate the SQL migrations. 
```

Comment:
```suggestion
2. Run `bunx drizzle-kit generate:pg` to generate the SQL migrations. 
```

Reply:
Not a typo, that command doesn't work with bun yet :(

Reply:
works on my machine:
```
 bunx drizzle-kit generate:pg
drizzle-kit: v0.20.14
drizzle-orm: v0.30.7

No config path provided, using default 'drizzle.config.ts'
Reading config file '/Users/lucas/vault/backend/drizzle.config.ts'
2 tables
balance_usdc 6 columns 0 indexes 0 fks
transfer_usdc 11 columns 0 indexes 0 fks

No schema changes, nothing to migrate 😴
```

Reply:
Good to know, I will change it to `bunx` then!